config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -60,23 +60,24 @@ spec:
                 same as the name returned by the CSI GetPluginName() call for that
                 driver. Required.
               type: string
-            snapshotClassName:
-              description: name of the SnapshotClass to which this snapshot belongs.
-              type: string
             source:
               description: source specifies from where a snapshot will be created.
                 This field is immutable after creation. Required.
               properties:
                 snapshotHandle:
-                  description: snapshotHandle specifies the CSI name of a pre-existing
+                  description: snapshotHandle specifies the "snapshot_id" of a pre-existing
```

Comment:
Should we specify this is the CSI snapshot_id? ditto everywhere

Reply:
I think the context should provide enough information?
I can add if that sound more clear. LMK

Reply:
yeah let's just quickly clarify that this is the CSI field since all the other fields we say which csi call it's from

Reply:
updated.